libr/anal/fcn.c

```diff
@@ -626,6 +626,14 @@ static bool isInvalidMemory(const ut8 *buf, int len) {
 	// return buf[0]==buf[1] && buf[0]==0xff && buf[2]==0xff && buf[3] == 0xff;
 }
 
+static bool isSymbolNextInstruction(RAnal *anal, RAnalOp *op) {
+	RFlagItem *fi = anal->flb.get_at (anal->flb.f, op->addr + op->size, false);
+	if (fi && (strstr (fi->name, "imp.") || strstr (fi->name, "sym.")
```

Comment:
you can just return the result of this if expression. drop the true/false things